server/jetstream_cluster_3_test.go

```diff
@@ -1315,3 +1316,60 @@ func TestJetStreamClusterHAssetsEnforcement(t *testing.T) {
 	})
 	require_Error(t, err, exceededErrs...)
 }
+
+func TestJetStreamClusterInterestStreamConsumer(t *testing.T) {
+	c := createJetStreamClusterExplicit(t, "R5S", 5)
```

Comment:
No need to change, but curious if this needs to be a 5 nodes to easily reproduce or it is just a copy paste from another test and this could have been a 3 nodes cluster?

Reply:
5 nodes showed it more repeatedly vs 3.